### PR TITLE
feat/with telemetry

### PR DESCRIPTION
**Pull request by kjuulh**
Initial telemetry functionality

### CODE REVIEW

File: pkg/telemetry/utility.go
Comment (mahlunar) [inline pkg/telemetry/utility.go:1]: Maybe rename to context.go?

File: shuttle.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ scripts:
     description: build code
     actions:
       - shell: go build
+      - shell: sleep 2
```
Comment (mahlunar) [inline shuttle.yaml:12]: remove?

**Review by mahlunar — COMMENTED** [review]
Good start!

I think we need some tests on the telemetry package. Do we want it exported? Maybe it should be move to internal instead of pkg?

What are your thoughts on the global variables you introduced? It does make it harder to test the integration to it, right?

File: shuttle.yaml (hunk 1 above)
Comment (kjuulh) [reply shuttle.yaml:12]: Some debug stuff I did, for testing longer durations in humio

Will remove xD

**Review by cablunar — APPROVED** [review]

**Pull request merged**